packages/terra-compact-interactive-list/src/CompactInteractiveList.jsx

```diff
@@ -70,12 +83,24 @@ const propTypes = {
   width: PropTypes.string,
 
   /**
-   * Columns minimum width should be a valid css string in value in px, em, or rem units..
+   * Callback function that is called when a selectable cell is selected. Parameters:
+   * @param {string} rowId rowId
+   * @param {string} columnId columnId
```

Comment:
You should return an object with named parameters for rowId and columnId instead of two parameters.

Reply:
I fixed the code to return the object in https://github.com/cerner/terra-framework/pull/1919/commits/fc32888e860735982a59eadfd00346f3b976688d

Reply:
I see you updated the code.  However, I think you also need to update the documentation.